Kentico/custom-element-samples
# Syntax Highlighter based on Ace editor

### Motivation

Existing issue: https://github.com/Kentico/custom-element-samples/issues/8

### How to test

See readme on how to use it.
Test for obtaining the value: https://codepen.io/milanlund/pen/rRZRVe
